Analytics/Tests/Infraestructure/PostHogServiceTests.swift

```diff
@@ -26,23 +26,32 @@ final class PosthogServiceTests: XCTestCase {
     }
 
     func testIdentify() {
-        XCTAssertNotNil(service.posthog)
-        XCTAssertTrue(service.posthog!.enabled)
+        guard let posthog = service.posthog else {
```

Comment:
If you want, there is a newish XCTUnwrap function you could use here that is made for this situation. You can use it like this: `let postHog = try XCTUnwrap(service.posthog)`